File: PUM.java
import java.util.Scanner;

public class PUM {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();
        int count =1;
        for(int i = 1; i <= num; i++){
            if(count % 4 != 0){
                System.out.print(count++ +" "+count++ +" "+count++ +" ");
                i--;
            }
            else{
                System.out.println("PUM");
                count++;
            }
        }        
    }
}

File: pum.cpp
#include<iostream>
using namespace std;
int main(){

int num;
int count = 1;
cin>>num;
for (int i = 0; i < num; i++)
{
    cout<<count++<<" "<<count++<<" "<<count++<<" ";
    if(count % 4 == 0){
        cout<<"PUM"<<"\n";
        count++;
    }
}

    return 0;
}